src/engraving/dom/engravingitem.cpp

```diff
@@ -175,6 +175,16 @@ EngravingItemList EngravingItem::childrenItems(bool all) const
     return list;
 }
 
+const std::shared_ptr<IEngravingConfiguration>& EngravingItem::configuration() const
+{
+    return score()->configuration.get();
```

Comment:
Is it right that the purpose here is to prevent making `EngravingItem`/`EngravingObject` `Injectable`, and only `Score`? (makes sense, just checking if I understand correctly)

Reply:
That's right!! Otherwise, we will have to pass a ioc context to each object... Although we are already passing the parent, and ultimately has access to a score.